MetaMaint Chems Nerf or Nothing

Zytolg opened the pull request:
I've been asked by @EOBGames to downgrade the chemistry setup in Metastation's Abandoned Medbay.

## Changelog
:cl:
qol: There's some eye candy in the Abandoned Medbay that now suggests that it was at some point a medbay. 
balance: Downgrades the Chem Dispenser and Heater to their Ghetto Counterparts
/:cl:

Zytolg committed:
Chems Nerf or Nothing

At the request of our Maptainer ~ GhettoMed

Capsandi commented on _maps/map_files/MetaStation/MetaStation.dmm:
```diff
@@ -12010,13 +12019,12 @@
-	},
-/turf/open/floor/plating,
+/obj/structure/frame/machine,
+/obj/item/circuitboard/machine/chem_dispenser,
```
```suggestion
/obj/item/circuitboard/machine/chem_master,
```
Did you mean to put a chem-master board in here(as to allow separation of chemicals)? A whole chem-dispenser makes the setup rival the pharmacy, and removes most of the ghetto in ghetto-chem.

Zytolg replied:
Thanks for the catch, I'll make this change right away.

Zytolg committed:
Investigates Integration Failure

Sleep deprived me took a look for wide awake me so that he won't have to deal with CI failure in the morning. The right object (ghetto chem heater) is being used here, but CI is failing on it for some reason so the problem is a problem. I don't recall these being mapped in the past either so that might explain why we see it now.

TheBonded commented:
While you're at it, as far as I know the peacekeeper plushie is still lacking a name

Zytolg commented:
This is currently waiting on #59580 to fix the bad overlays with Ghetto-ized Heaters.

EOBGames commented:
Applying the red label for now, until #59580 is merged

Zytolg committed:
Merge branch 'master' into ghettoMetaMedNerf
